src/components/stereo/stereo.js

```diff
@@ -0,0 +1,85 @@
+//@flow
+import style from '../../styles/style.scss';
+import {h} from 'preact';
+import {Localizer, Text} from 'preact-i18n';
+import BaseComponent from '../base';
+import {default as Icon, IconType} from '../icon';
+import {KeyMap} from "../../utils/key-map";
+import {actions as stereoActions} from "../../reducers/stereo";
+import {bindActions} from "../../utils/bind-actions";
+import {connect} from "preact-redux";
+import {actions} from "../../reducers/shell";
+
+/**
+ * mapping state to props
+ * @param {*} state - redux store state
+ * @returns {Object} - mapped state to this component
+ */
+const mapStateToProps = state => ({
+  stereoMode: state.stereo.stereoMode
+});
+
+@connect(mapStateToProps, bindActions(Object.assign({}, actions, stereoActions)))
+/**
+ * StereoControl component
+ *
+ * @class StereoControl
+ * @example <StereoControl/>
+ * @extends {BaseComponent}
+ */
+class StereoControl extends BaseComponent {
+  /**
+   * @static
+   * @type {string} - Component display name
+   */
+  static displayName = 'stereo';
+  /**
+   * Creates an instance of StereoControl.
+   * @memberof StereoControl
+   */
+  constructor() {
+    super({name: 'Stereo'});
+  }
+
+  /**
+   * Stereo click handler
+   *
+   * @returns {void}
+   * @memberof StereoControl
+   */
+  onClick(): void {
+    this.notifyClick();
```

Comment:
see https://github.com/kaltura/kaltura-player-js/pull/130/files#r196646166 and https://github.com/kaltura/playkit-js/pull/239/files#r196647201 use `player.toggleVrStereoMode()`
Also need to be reactive to VR events, so need to add a `vrStereoModeChanged` event (when change is done via API of player) and listen to it and change state accordingly.